package.json

```diff
@@ -22,7 +22,7 @@
     "build:chatbot": "nx run chatbot:build",
     "build:parsers": "nx run parsers:build",
     "build:vscode-react": "nx run vs-code-react:build",
-    "build:vscode-designer": "nx run vs-code-designer:build && nx run vs-code-data-mapper-react:build && nx run vs-code-designer-react:build && nx run vs-code-react:build",
+    "build:vscode-designer": "nx run vs-code-designer:build && nx run vs-code-react:build",
     "build:vscode-datamapper": "nx run vs-code-designer:build && nx run vs-code-data-mapper-react:build",
```

Comment:
Reminder: remove this when removing the vs-code-data-mapper-react and vs-code-designer-react folders